### PR TITLE
fix(logo): update 9X to new design #174

### PR DESCRIPTION
**Pull request by blackfalcon**
Changes to be committed:
modified:   src/icons/logos/tail-9X.svg

# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #174 

## Summary:

This PR will update the 9X tail to the new design. 

## Type of change:

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Update tail design

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/data/logoIcons.json [hunk 1]
```diff
@@ -302,7 +302,7 @@
     {
       "name": "tail-9X",
       "title": "Airplane tail image for Mokulele Airlines",
-      "viewBox": "0 0 856 830"
+      "viewBox": "0 0 736 707"
```
Comment (shonore-aa) [inline src/data/logoIcons.json:305]: The Added image doesn't show anything.

Comment (jason-capsule42) [inline src/data/logoIcons.json:305]: All other tail logos have the `0 0 856 830` value. Why is this the only tail that is different?

Comment (jason-capsule42) [reply src/data/logoIcons.json:305]: > The Added image doesn't show anything.

Concur here - when I load the SVG in this PR into https://www.svgviewer.dev/ it doesn't throw an error but the content is blank.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/data/logoIcons.json (hunk 1 above)
Comment (blackfalcon) [reply src/data/logoIcons.json]: That was my bad. I fixed the SVG data but didn't push the branch. Oops.

Comment (blackfalcon) [reply src/data/logoIcons.json]: The SVG added to the repo does not have attributes. This is all added in the build process via the data json file for the SVGs added to the `./dist` dir. 

A directory of icons, logos and tails can be seen here alaska.iconstails.surge.sh

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**